src/containers/new-dao.js

```diff
@@ -35,6 +38,10 @@ const mapDispatchToProps = (dispatch, ownProps) => {
     saveDao: (dao) => {
       dispatch(saveDao(dao))
     },
+    saveSemBalanceFunc: async(web3) => {
```

Comment:
No need to add "Func" to this function name- its not going to collide with the action name inside.